locale/en/blog/vulnerability/july-2022-security-releases.md

```diff
@@ -15,6 +15,9 @@ releases lines on or shortly after Tuesday, July 5th, 2022 in order to address:
 * Three medium severity issues.
 * Two high severity issues.
 
+**Note**: this release will include the latest OpenSSL Security Release. Therefore, a potential delay is acceptable
```

Comment:
I think we should be more specific.  I'd suggest do we something like we have done in the past (https://github.com/nodejs/nodejs.org/blob/88f8ca74c67b439aca43b456162694028622bd35/locale/en/blog/vulnerability/openssl-and-low-severity-fixes-jan-2016.md#update-30-jan-2016-release-postponement) and add a separate new section at the top announcing the delay.

I'm thinking in the initial update we should say that we will evaluate the OpenSSL changes when they are available on Tuesday and then announce our new target date by Thursday or ealier?.

# Update

Reply:
Makes sense. I'll update